templates/pages/auth/create-account.html

```diff
@@ -20,27 +20,26 @@ <h1 class="page-heading">{{lang 'create_account.heading' }}</h1>
                     {{inject 'state_error' (lang 'errors.state_error')}}
                 </div>
                 {{#if forms.create_account.recaptcha.enabled }}
-                <div class="panel">
-
-                    <header class="panel-header">
-                        <h3 class="panel-title">{{lang 'create_account.captcha_heading' }}</h3>
-                    </header>
-                    <div class="panel-body">
-                        <script type="text/javascript"
-                                src="https://www.google.com/recaptcha/api/challenge?k={{forms.create_account.recaptcha.public_key}}">
-                        </script>
-                        <noscript>
-                            <iframe src="https://www.google.com/recaptcha/api/noscript?k={{forms.create_account.recaptcha.public_key}}" height="300" width="500" frameborder="0"></iframe>
-                            <br>
-                            <textarea name="recaptcha_challenge_field" rows="3" cols="40">
-                            </textarea>
-                            <input type="hidden" name="recaptcha_response_field" value="manual_challenge">
-                        </noscript>
+                    <div class="panel">
+                        <header class="panel-header">
+                            <h3 class="panel-title">{{lang 'create_account.captcha_heading' }}</h3>
+                        </header>
+                        <div class="panel-body">
+                            <script type="text/javascript"
+                                    src="https://www.google.com/recaptcha/api/challenge?k={{forms.create_account.recaptcha.public_key}}">
+                            </script>
+                            <noscript>
+                                <iframe src="https://www.google.com/recaptcha/api/noscript?k={{forms.create_account.recaptcha.public_key}}" height="300" width="500" frameborder="0"></iframe>
+                                <br>
+                                <textarea name="recaptcha_challenge_field" rows="3" cols="40">
+                                </textarea>
+                                <input type="hidden" name="recaptcha_response_field" value="manual_challenge">
+                            </noscript>
+                        </div>
                     </div>
-                </div>
                 {{/if}}
                 <div class="form-actions">
-                    <input type="submit" class="button button--primary" value="{{lang 'forms.create_account.submit_value' }}" />
+                    <input type="submit" class="button button--primary" value="{{lang 'forms.create_account.submit_value' }}"  {{#unless settings.account_creation_enabled}}disabled{{/unless}} />
```

Comment:
why not disable the entire form?
People will think is a bug if you only disable the the submit button